content/webapp/services/prismic/fetch/articles.ts

```diff
@@ -136,32 +136,6 @@ export const graphQuery = `{
           caption
         }
       }
-      ...on soundcloudEmbed {
-        non-repeat {
-          iframeSrc
-        }
-      }
-      ...on vimeoVideoEmbed {
-        non-repeat {
-          embed
-        }
-      }
-      ...on instagramEmbed {
-        non-repeat {
-          embed
-        }
-      }
-      ...on twitterEmbed {
-        non-repeat {
-          embed
-        }
-      }
-      ...on youtubeVideoEmbed {
-        non-repeat {
-          embed
-          caption
-        }
-      }
```

Comment:
sweet, these GraphQL queries have always felt a bit mysterious, good to see them gradually being trimmed!